service/gradle/dependencies.gradle

```diff
@@ -37,6 +37,11 @@ dependencies {
     exclude group: "org.broadinstitute.dsde.workbench", module: "sam-client_2.12"
   }
 
+  //Terra Landing Zone Service
+  implementation ('bio.terra:terra-landing-zone-service:0.0.1-ALPHA-7-SNAPSHOT') {
```

Comment:
I will replace it with release version once corresponding PR in LZ library is merged.